pkg/landscaper/installations/operation.go

```diff
@@ -172,7 +172,7 @@ func (o *Operation) GetImportedDataObjects(ctx context.Context) (map[string]*dat
 
 		var (
 			sourceRef *lsv1alpha1.ObjectReference
-			configGen = strconv.Itoa(int(do.Raw.Generation))
+			configGen = do.Metadata.Hash
```

Comment:
You changed this only for `GetImportedDataObjects`, but `GetImportedTargets` follows the same logic - shouldn't this be changed there too?

Reply:
no because for targets it is still valid to use the generation field.
Only secrets and configmaps do not work with it

Reply:
ah, ok, then it should be fine.